### ISSUE TITLE
Why met nan in training your iterdet faster rcnn model

### ISSUE THREAD

**Issue by qgh1223** [saic-vul/iterdet#3]
Why met nan in training your iterdet faster rcnn model?The learning rate is too big?

**Comment by qgh1223** [comment]
I met a series of very strange training loss Intermittent loss appears nan like this
2020-05-18 09:24:01,065 - mmdet - INFO - Start running, host: root@1be15257c199, work_dir: /content/iterdet/work_dirs/iterdet/crowd_human_full_faster_rcnn_r50_fpn_2x
2020-05-18 09:24:01,065 - mmdet - INFO - workflow: [('train', 1)], max: 50 epochs
2020-05-18 09:24:48,902 - mmdet - INFO - Epoch [1][50/1084]	lr: 0.01000, eta: 14:19:58, time: 0.953, data_time: 0.011, memory: 4370, loss_rpn_cls: 0.5577, loss_rpn_bbox: 0.2245, loss_cls: 0.1869, acc: 92.8633, loss_bbox: nan, loss: nan
2020-05-18 09:25:37,095 - mmdet - INFO - Epoch [1][100/1084]	lr: 0.01000, eta: 14:24:34, time: 0.965, data_time: 0.010, memory: 4940, loss_rpn_cls: 0.3339, loss_rpn_bbox: 0.2323, loss_cls: 0.1757, acc: 93.4844, loss_bbox: nan, loss: nan
2020-05-18 09:26:23,531 - mmdet - INFO - Epoch [1][150/1084]	lr: 0.01000, eta: 14:14:45, time: 0.929, data_time: 0.009, memory: 4940, loss_rpn_cls: 0.2372, loss_rpn_bbox: 0.2577, loss_cls: 0.2204, acc: 91.3359, loss_bbox: 0.4434, loss: 1.1586
2020-05-18 09:27:12,087 - mmdet - INFO - Epoch [1][200/1084]	lr: 0.01000, eta: 14:18:53, time: 0.971, data_time: 0.008, memory: 4940, loss_rpn_cls: 0.1980, loss_rpn_bbox: 0.2456, loss_cls: 0.2006, acc: 91.9141, loss_bbox: 0.4835, loss: 1.1277
2020-05-18 09:28:01,634 - mmdet - INFO - Epoch [1][250/1084]	lr: 0.01000, eta: 14:24:36, time: 0.990, data_time: 0.009, memory: 5229, loss_rpn_cls: 0.1688, loss_rpn_bbox: 0.2417, loss_cls: 0.2449, acc: 89.6172, loss_bbox: 0.5452, loss: 1.2006
2020-05-18 09:28:50,660 - mmdet - INFO - Epoch [1][300/1084]	lr: 0.01000, eta: 14:26:47, time: 0.982, data_time: 0.010, memory: 5229, loss_rpn_cls: 0.1298, loss_rpn_bbox: 0.2240, loss_cls: 0.2087, acc: 91.7461, loss_bbox: 0.4504, loss: 1.0129
2020-05-18 09:29:37,285 - mmdet - INFO - Epoch [1][350/1084]	lr: 0.01000, eta: 14:21:45, time: 0.932, data_time: 0.008, memory: 5631, loss_rpn_cls: 0.1381, loss_rpn_bbox: 0.2442, loss_cls: 0.2066, acc: 91.4336, loss_bbox: 0.4908, loss: 1.0798
2020-05-18 09:30:24,746 - mmdet - INFO - Epoch [1][400/1084]	lr: 0.01000, eta: 14:19:44, time: 0.949, data_time: 0.009, memory: 5631, loss_rpn_cls: 0.1376, loss_rpn_bbox: 0.2086, loss_cls: 0.1848, acc: 92.6289, loss_bbox: 0.3825, loss: 0.9134
2020-05-18 09:31:12,416 - mmdet - INFO - Epoch [1][450/1084]	lr: 0.01000, eta: 14:18:15, time: 0.952, data_time: 0.009, memory: 6048, loss_rpn_cls: 0.1201, loss_rpn_bbox: 0.1902, loss_cls: 0.1584, acc: 93.6094, loss_bbox: nan, loss: nan
I have used SGD and Adam and reduce learning rate.It still happens.
I haven't meet this when using mmdetection before.It is a new feature in new mmdetection after update on May?

**Comment by filaPro** [comment]
I didn't meet this behavior. Are you training on CrowdHuman dataset with exactly the same [config](https://github.com/saic-vul/iterdet/blob/master/configs/iterdet/crowd_human_full_faster_rcnn_r50_fpn_2x.py) as in this repo? And CrowdHuman dataset is preprocessed with our [tool](https://github.com/saic-vul/iterdet/blob/master/tools/convert_datasets/crowd_human.py)?

**Comment by qgh1223** [comment]
I don't use the crowdhuman dataset.I want to use this algorithm in another dataset

**Comment by filaPro** [comment]
In this case it's really hard to figure out the problem. Can you share your config? Did you remove images with 0 objects per image from your dataset?
